sklearn/utils/testing.py

```diff
@@ -58,7 +58,6 @@
 
 from sklearn.base import (ClassifierMixin, RegressorMixin, TransformerMixin,
                           ClusterMixin)
-from sklearn.cluster import DBSCAN
```

Comment:
Grr... Yes, this is a problem of our CI running flake8 only on diffs. I've removed this import in master.